sonar-plugin-api/src/main/java/org/sonar/api/utils/log/DefaultProfiler.java

```diff
@@ -94,24 +94,31 @@ public Profiler startInfo(String message) {
 
   @Override
   public Profiler stopTrace() {
-    return doStopWithoutMessage(LoggerLevel.TRACE);
+    return doStopWithoutMessage(LoggerLevel.TRACE, " (done)");
   }
 
   @Override
   public Profiler stopDebug() {
-    return doStopWithoutMessage(LoggerLevel.DEBUG);
+    return doStopWithoutMessage(LoggerLevel.DEBUG, " (done)");
   }
 
   @Override
   public Profiler stopInfo() {
-    return doStopWithoutMessage(LoggerLevel.INFO);
+    return stopInfo(false);
   }
+  
 
-  private Profiler doStopWithoutMessage(LoggerLevel level) {
+  @Override
+  public Profiler stopInfo(boolean cacheUsed) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Provide multiple methods instead of using "cacheUsed" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2301)